### PR TITLE
capitalize also emails

### LINKED ISSUE #629: Capitalization of proper names
@sfkeller noted:

* We still have occurences of "Osmaxx" instead of "OSMaxx" in user-visible strings (e.g. in the `<title>` HTML element)
* Esri, although being an acronym, has switched to capitalize their name "Esri" instead of the previous "ESRI".
* "Garmin" should be used instead of "GARMIN" or "garmin" (the latter is currently used on the GIS options title)

Use the current official capitalization everywhere. (For user-visible strings.)

### PR DESCRIPTION
**Pull request by hixi**
Reviewed by:
- [x] @das-g 

Closes #629

### CODE REVIEW

File: docs/development/project-development-environment.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ The following list ist supported:
     - DJANGO_EMAIL_HOST_PASSWORD=''
     - DJANGO_EMAIL_HOST_USER=''
     - DJANGO_EMAIL_PORT=25
-    - DJANGO_EMAIL_SUBJECT_PREFIX='[OSMAXX-DEV] '
+    - DJANGO_EMAIL_SUBJECT_PREFIX='[OSMaxx-dev] '
```
Comment (das-g) [inline docs/development/project-development-environment.md]: Good find! :+1:

File: osmaxx/utilities/shortcuts.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def inform_mail(self, subject, mail_body, warn_if_no_email=True):
         try:
             email_address = self.recipient.email
             mail.send_mail(
-                '[OSMAXX] ' + subject,
+                '[OSMaxx] ' + subject,
```
Comment (das-g) [inline osmaxx/utilities/shortcuts.py]: Should `settings.EMAIL_SUBJECT_PREFIX` be used here?

Comment (hixi) [inline osmaxx/utilities/shortcuts.py]: either that, or remove the setting.

Comment (das-g) [inline osmaxx/utilities/shortcuts.py]: I'd lean towards using the setting here. Then, if we ever use [`mail_admins()`](https://docs.djangoproject.com/en/1.9/topics/email/#mail-admins) or [`mail_managers()`](https://docs.djangoproject.com/en/1.9/topics/email/#mail-managers), (where Django would make use of the setting automatically), the setting is already in place.

Comment (hixi) [inline osmaxx/utilities/shortcuts.py]: Then let's do that.

**Pull request merged**